puma/hlplots/tagger.py

```diff
@@ -37,6 +41,33 @@ def __post_init__(self):
     def __repr__(self):
         return f"{self.name} ({self.label})"
 
+    def is_flav(self, flavour: Flavour | str):
+        """Return indices of jets of given flavour.
+
+        Parameters
+        ----------
+        flavour : str
+            Flavour to select
+
+        Returns
+        -------
+        np.ndarray
+            Array of indices of the given flavour
+        """
+        flavour = Flavours[flavour] if isinstance(flavour, str) else flavour
+        return flavour.cuts(self.labels).idx
+
+    @property
+    def output_nodes_lower(self):
+        """Return the lowercase output nodes.
+
+        Returns
+        -------
+        list
+            List of lower case output nodes
+        """
+        return [x.name.lower() for x in self.output_nodes]
+
     @property
     def probailities(self):
```

Comment:
should we fix this typo?
```suggestion
    def probabilities(self):
```

Reply:
this will also warrant changes in other locations, but I think it's better to fix this now.

Reply:
done!